lib/generator.dart

```diff
@@ -12,15 +15,9 @@ import './generator/errors.dart';
 import './generator/graphql_helpers.dart' as gql;
 import './generator/helpers.dart';
 import './schema/options.dart';
-import 'visitor.dart';
 
 typedef _OnNewClassFoundCallback = void Function(Context context);
 
-void _log(Context context, int align, Object logObject) {
-  if (!context.log) return;
```

Comment:
The thing about having this `log` field on context to ignore logs is when we're don't care about (recursive) type generation on visitors (just like [here](https://github.com/comigor/artemis/pull/207/files#diff-139be2e2838213da1d6734bbd32c45d6R597)). If we remove this, we won't be able to control the log tree and it'll be even harder to debug the generation (at least for me) :(

Reply:
If it is valuable feature for you I'll revert changes back)
I just thought that control over `verbose` is enough but totally forgot about granularity of control